plugin-server/tests/main/db.test.ts

```diff
@@ -135,6 +136,46 @@ describe('DB', () => {
         })
     })
 
+    describe('updatePerson', () => {
+        it('Clickhouse and Postgres are in sync if multiple updates concurrently', async () => {
```

Comment:
Note the test isn't testing this - it's testing whether kafka producer is called correctly. This means we're lacking an integration test for this.

Reply:
how should it be named differently then, as yes it's a unit test not an integration test.